### DISCUSSION TITLE
Does Event counters in SqlClient support multi-instance counters similar to Performance counters?

### DISCUSSION THREAD

**Discussion by Trivedi-Ashish** [dotnet/SqlClient#1545] [Q&A] [answered]
I was wondering if event counters in SqlClient support multi-level? 
By multi-level, I mean if there are multiple connection pools in the application using SqlClient then in this scenario, are there any event counters displayed per connection pool?
For e.g. If for instance we talk about "**number-of-active-connections**" event counter, will there be multiple instances of this counter available, one for each connection pool in a multiple connection pools scenario?
@cheenamalhotra

**Comment by DavoudEshtehari** [comment, marked as answer]
Hi @Trivedi-Ashish,
These two counters are similar, there is no multi-level feature.

**Reply by Trivedi-Ashish** [reply]
Just to clarify,
If we need to know number of free or active connections in a particular pool given that there might be multiple pools,
there is no way to query that?

**Reply by DavoudEshtehari** [reply]
I'm afraid not. The counters have been designed to have the lowest impact on app performance.
